crates/adapters/src/integrated/delta_table/input.rs

```diff
@@ -210,9 +217,186 @@ impl DeltaTableInputEndpointInner {
         }
     }
 
+    /// Load the entire table snapshot as a single "select * where <filter>" query.
```

Comment:
always *? Sometimes it's useful to drop some columns too.

Reply:
We could specify column list, but columns that are not defined in the Feldera table are dropped by default.

Reply:
dropped may not be the same as not read

Reply:
True. That could be a useful optimization.